pages/media.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title> ORA UI </title>
+    <link href="../libs/font-awesome/css/font-awesome.css" rel="stylesheet">
+    <link href="../libs/Ionicons/css/ionicons.css" rel="stylesheet">
+    <link rel="stylesheet" href="../css/magen-iot-admin.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
use the local library instead. you can see how it's done in the other pages

Reply:
Others are same. Except bootstrap line isnt there

Reply:
> Others are same. Except bootstrap line isnt there

the files link to bootstrap by '<link rel="stylesheet" href="../css/bootstrap.min.css">' instead of the cdn

Reply:
oh arigato <3